pkg/core/native/native_test/neo_test.go

```diff
@@ -271,6 +271,35 @@ func TestNEO_Vote(t *testing.T) {
 		require.NotEqual(t, candidates[0], pubs[i])
 		require.NotEqual(t, candidates[len(candidates)-1], pubs[i])
 	}
+	//LastGasPerVote should be 0 after unvoting
```

Comment:
s/`//LastGasPerVote should be 0 after unvoting`/`// LastGasPerVote should be 0 after unvoting.`